.gitignore

```diff
@@ -5,3 +5,6 @@ node_modules
 .DS_Store
 .vscode
 .node_modules_git
+test/mocha/workspaces/test-workspaces/default/ContractCache
```

Comment:
could we just make this `ContractCache` or `**/ContractCache`?

Reply:
Probably. There are more and they are annoying and I don't know what is making them.

Reply:
Ganache 2.0 makes them on opening the workspace